test/suites/secondary_redis.bash

```diff
@@ -0,0 +1,63 @@
+SUITE_secondary_redis_PROBE() {
+     if [ "`which redis-server`" = "" ]; then
+        echo "no redis-server"
+        return
+    fi
+    if [ "`which redis-cli`" = "" ]; then
+        echo "no redis-cli"
+        return
+    fi
+}
+
+SUITE_secondary_redis_SETUP() {
```

Comment:
My http storage backend also spawns a server in the `_SETUP` function. Unfortunately this will spawn multiple servers for multiple `TEST` cases. I believe we need a `_TEARDOWN` method.

Reply:
Did a workaround, i.e. copied it to the main test instead. Also for some reason it _never_ shut down for the Mac builds ?